### ISSUE TITLE
Dynamic Entry Titles

### ISSUE THREAD

**Issue by jasonvarga** [statamic/ideas#78]
Title shouldn't really need to be required. It could be generated based on other fields, defined on the collection level.

https://github.com/statamic/v2-hub/issues/1920

**Comment by jackmcdade** [comment]
Moved this to the ideas repo.

**Comment by emil-nasso** [comment]
I agree. I'm evaluating using statamic as a headless api for content for various frontends. Neither title or slug makes sense for many of the content types we have. All access to this content is via the graphql api, there will never be a single HTML-view.
